src/main/java/frc/robot/RobotContainer.java

```diff
@@ -79,10 +82,25 @@ public RobotContainer()
 
   private void configureBindings() {
    
+    //Driver Controls
+    /*
+     * Moves:
+     *  - Swerve
+     *  - Floor Intake + conveyor maybe? 
+     *  - Climb?
+     */
     m_driverController.x().onTrue((new InstantCommand(drivebase::zeroGyro)));
     m_driverController.y().whileTrue(new ConveyInwardCommand(conveyor));
     m_driverController.a().whileTrue(new FI_IntakeForward(floorIntake));
     m_driverController.b().debounce(.1).onTrue(new ArmPositions(arm));
+
+    //Gunner Controls
+    /*
+     *  Moves:
+     *   - Fly Wheel
+     *   - Pivot 
+     */
+    m_coDriverController.button(1); 
```

Comment:
What's the purpose of this line here?

Reply:
just to see if implementing just a button would work, should I get rid of it, it was a tester.

Reply:
Yeah, in general if you can remove test-only code, it'll keep your codebase cleaner

Reply:
sounds good, sweeping up the trash 🧹